README.md

```diff
@@ -4,16 +4,15 @@
 
 Scale is a simulation of modern day society, from the bottom-up. 
 Each individual has its own thought model, meaning every action has its importance and influences the environment.  
-Scale is not a video game, more of a zero-player game like the game of life. A little interaction might be present but will be very limited.  
-Thus, lot of power can be invested into making Scale a unique persistent world,
+Scale is not a video game, but more of a zero-player game (like the game of life), in which there might be a small and very limited interaction.  
+Thus, a lot of power could be invested into making Scale a unique persistent world,
 where attention to detail is achievable. 
 
 ## How ?  
 This is of course very ambitious, so inspired by the WASM project,
 a minimal viable product should be made where features are increasingly added.
 
-The first [milestone](https://github.com/Uriopass/Scale/projects/1) is to be able to have humans having homes and a workplace,
-using their feet or their cars to go there.  
+The first [milestone](https://github.com/Uriopass/Scale/projects/1) is to be able to bring humans to the world. They would have their own homes and a workplace, for which they could use their cars or simply walk to go there.  
```

Comment:
> for which  

For which est un peu dégeu comme construction

Reply:
Je trouve ça déjà un peu plus pro que "is to be able to have"

Reply:
Tant qu'a changer quelque chose, autant le changer en bien. L'objectif d'une PR n'est pas de passer de nul à moins nul, mais de nul à bien.

Reply:
nul à moins nul c'est déjà mieux, et le principe des commits github est de faire avancer petit à petit le projet, en des milliers d'étapes